src/js/components/album-preview.component.js

```diff
@@ -0,0 +1,31 @@
+import Component from "../models/component.model.js";
+const baseURL = `${window.location.pathname.split("/")[0]}/posts-albums/src/pages`;
+
+export default class AlbumPreview extends Component {
+    constructor(userId, id, title, photos) {
+        super();
+        this.userId = userId;
+        this.id = id;
+        this.title = title;
+        this.photos = photos;
+    }
+
+    get template() {
+        return `<div class="album" id="album-${this.id}">
+                    <div class="album__header">
+                        <div class="album__title">${this.title}</div>
+                        <div class="album__user" id="user-${this.userId}">
+                            <img src="../images/abstract-user-flat-1.svg" alt="user"/>
+                        </div>
+                    </div>
+                    <a href="${baseURL}/album.html?id=${this.id}">
```

Comment:
url should be `/src/pages/album.html?id=${this.id}`, otherwise it doesn't work
<img width="1110" alt="Screen Shot 2022-02-03 at 13 10 58" src="https://user-images.githubusercontent.com/56446932/152313190-5239183e-0d4a-4496-8cb8-43bcb92aa678.png">